client/index.html

```diff
@@ -801,6 +811,15 @@ <h1 class="title">Help</h1>
 
 						<div class="help-item">
 							<div class="subject">
+								<code>/unban user</code>
```

Comment:
https://media.giphy.com/media/Kh296M49Zr1Sg/giphy.gif

Goodness, it's clearly too early.